src/components/v2/Tooltip/Tooltip.tsx

```diff
@@ -3,7 +3,7 @@ import { Global } from '@emotion/react';
 import MuiTooltip, { TooltipProps } from '@mui/material/Tooltip';
 import { useStyles } from './styles';
 
-type ITooltipProps = TooltipProps;
+export type ITooltipProps = TooltipProps;
```

Comment:
shouldn't be an prefaced with an `I` as it is not an interface. There is an eslint plugin for this. 
I also updated this to take an element so `<Trans>` can be used here

Reply:
Good point about the interface.

About the tooltip, we should pass the full text directly otherwise the parser can't extract the translation keys.